R/pkg/R/RDD.R

```diff
@@ -306,17 +306,26 @@ setMethod("checkpoint",
 #'\dontrun{
 #' sc <- sparkR.init()
 #' rdd <- parallelize(sc, 1:10, 2L)
-#' numPartitions(rdd)  # 2L
+#' getNumPartitions(rdd)  # 2L
 #'}
-#' @rdname numPartitions
+#' @rdname getNumPartitions
+#' @aliases getNumPartitions,RDD-method
+#' @noRd
+setMethod("getNumPartitions",
+          signature(x = "RDD"),
+          function(x) {
+            callJMethod(getJRDD(x), "getNumPartitions")
+          })
+
+#' numPartitions is the same as getNumPartitions
```

Comment:
In fact, if we are deprecating, consider adding
```
.Deprecated(newFuncSig, old = oldFuncSig)
```
call to the deprecated version